src/App.tsx

```diff
@@ -142,7 +142,7 @@ function App(): React.JSX.Element {
                         path={credentialPageRoute}
                         element={
                             <OidcSecure>
-                                <Protected allowedRole={'user'}>
+                                <Protected allowedRole={'all'}>
```

Comment:
we must allow here only exactly user and isv roles. We can change the allowedRole paramter to an array if needed.